app/components/ProgressStepIndicator.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {Col, Row} from 'react-bootstrap';
+import {Variant} from 'react-bootstrap/esm/types';
+
+interface NumberCircleProps {
+  number: number;
+  style: Variant;
+}
+
+interface Props {
+  steps: Array<{
+    number: number;
+    description: string;
+    badgeStyle: Variant;
+  }>;
+}
+
+const NumberedCircle: React.FunctionComponent<NumberCircleProps> = ({
+  number,
+  style
+}) => {
+  return (
+    <>
+      <div className={`numberedCircle badge-${style}`}>{number}</div>
+      <style jsx>{`
+        .numberedCircle {
+          margin-top: -1.75rem;
+          width: 2.44rem;
+          line-height: 2.44rem;
+          border-radius: 50%;
+          text-align: center;
+          font-weight: bold;
+        }
+      `}</style>
+    </>
+  );
+};
+
+export const ProgressStepIndicator: React.FunctionComponent<Props> = ({
+  steps
+}) => {
+  return (
+    <>
+      <div className="progress ml-1 mr-1">
+        <div
+          role="progressbar"
+          className="progress-bar"
+          aria-valuenow={0}
+          aria-valuemin={0}
+          aria-valuemax={100}
+          aria-label="operator-progress-indicator"
+          style={{width: '0%'}}
+        />
+      </div>
```

Comment:
This could be refactored in a separate class - not sure if we need it at this moment